bindata/network/ovn-kubernetes/common/008-script-lib.yaml

```diff
@@ -589,6 +589,19 @@ data:
         ovn_v6_join_subnet_opt="--gateway-v6-join-subnet {{.V6JoinSubnet}}"
       fi
 
+{{- if not .OVNIPsecEnable }}
+      # flush out xfrm state and policy entries so that it cleans up
+      # entries created for ovs. Since it's cleaning up whole entries,
+      # it may cause slight interruption in case of external ipsec
+      # configuration.
+      echo "remove ovs ipsec state if present"
```

Comment:
problem is, this also removes the N-S state, not just ovs
so need to verify OVNIPsecEnable actually means both?

Reply:
yes, it also does ipsec service restart and wouldn't this get ipsec state for NS back into the system ?
Moreover this logic is similar to what ovn-ipsec container does [here](https://github.com/openshift/cluster-network-operator/blob/master/bindata/network/ovn-kubernetes/common/ipsec-host.yaml#L231-L238)  and same thing happens for container restart scenarios as well. 

The `OVNIPsecEnable` flag is used only to identify if EW IPsec is enabled or not.

The IPsec mode is changed in following order `Full -> Disabled -> External`, EW ipsec connections in `openshift.conf ` occasionally not cleaned up even after IPsec disabled, then external mode gets back those connections active again.  we may delete only `openshift.conf `file to handle this case,  but not sure about the behavior for `Full -> External` . Hence this change is introduced to handle all the cases.